src/referencePackages/src/system.security.principal.windows/4.7.0/System.Security.Principal.Windows.4.7.0.csproj

```diff
@@ -4,12 +4,12 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard1.3;netstandard2.0;netcoreapp3.0;net46;net461</TargetFrameworks>
     <NuspecFile>$(ArtifactsBinDir)system.security.principal.windows/4.7.0/system.security.principal.windows.nuspec</NuspecFile>
+    <DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>
```

Comment:
(nit) for CompilerServices.Unsafe, you have this in the 2nd PropertyGroup

Reply:
Fixed